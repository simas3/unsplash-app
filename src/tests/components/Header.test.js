import React from 'react'
import Header from '../../components/Header'
import { configure, shallow } from 'enzyme';
import toJSON from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

test('should render Header', () => {
    const wrapper = shallow(<Header />)
    expect(toJSON(wrapper)).toMatchSnapshot()
    expect(wrapper.find('h1').length).toBe(1)
    expect(wrapper.find('h1').text()).toEqual('Unsplash App')
})