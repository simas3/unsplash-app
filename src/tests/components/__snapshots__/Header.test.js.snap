// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header 1`] = `
<div
  className="header"
>
  <div
    className="container"
  >
    <h1
      className="header__title"
    >
      Unsplash App
    </h1>
  </div>
</div>
`;
